html/css/style.css

```diff
@@ -1,54 +1,218 @@
+    margin: 60px 100px;
     box-shadow: 5px 8px 8px #888888;
-  }
-.container--header {
+}
+.co-block--upper {
     display: flex;
-    height: 35px;
+    height: 4.5%;
 }
-.container--lower {
+.co-block--lower {
+    border-top: 0.5px solid lightgray;
     display: flex;
-    border-top: 1px solid lightgray;
-    height: 35px;
 }
-#checkout {
-    font-weight: bold;
-    flex: 6;
+.header--title {
+    flex: 7;
+    padding: 4px;
 }
-#shipping {
+.header--list {
     flex: 25;
-    padding: 10px;
+    padding: 4px;
 }
-#icon {
+.header--icon__close {
     flex: 0.5;
     align-items: center;
+    padding: 10px
 }
-#iconId {
-    padding: 10px;
+.breadcrumbs li:after {
+    padding: 15px;
+    content: "/";
+    color: grey;
+}
+.breadcrumbs li:last-child:after {
+    content: "";
+}
+.breadcrumbs {
+    padding: 7px;
+}
+
+.breadcrumbs li{
+    display: inline;
 }
 
-#payment {
-    flex-grow: 3;
+.header--title__checkout {
+    font-weight: bold;
+}
+.breadcrumbs--active {
+    font-weight: bold;
+}
+.block--lower__payment {
+    height: 100%;
+    flex: 3;
+}
+.block--lower__cart {
+    height: 100%;
+    flex: 2;
+    display: flex;
+    flex-direction: column;
+    background-color: #F7F7F7;
 }
-#cart {
-    flex-grow: 2;
-    background: lightgray;
+h1 {
+    padding: 7px;
 }
-#list li:after {
+.image--cards {
+    width: 90%;
     padding: 5px;
-    content: "/";
-    color: grey;
+    
 }
-#list li:last-child:after {
-    content: "";
+.card-details--date-cvv{ 
+    display: flex;
 }
-#list li{
-    display: inline;
+#card-name {
+    border: 0.5px solid lightgray;
+    padding: 4px;
+    border-radius: 4px;
+    width: 90%;
+    height: 40px;
+    margin: 5px;
+    font-size: 12px;
+}
+#card-number {
+    border: 0.5px solid lightgray;
+    border-radius: 4px;
+    padding: 4px;
+    width: 90%;
+    height: 40px;
+    margin: 5px;
+    font-size: 12px;
+}
+#card-date {
+    border: 0.5px solid lightgray;
+    border-radius: 4px;
+    padding: 4px;
+    width: 45%;
+    height: 40px;
+    margin: 5px;
+    font-size: 12px;
+}
+#card-cvv {
+    border: 0.5px solid lightgray;
+    border-radius: 4px;
+    padding: 4px;
+    width: 45%;
+    height: 40px;
+    margin: 5px;
+    font-size: 12px;
+}
+#card-submit {
+    border-radius: 4px;
+    padding: 4px;
+    width: 90%;
+    height: 40px;
+    margin: 5px;
+    color: white;
+    border: 0.5px solid  #4CAF50;
+    background-color: #4CAF50;
+    font-size: 12px;
+}
+.card-details{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.card-details div {
+    width: 45%;
+}
+hr {
+    width: 95%;
+}
+.image--products {
+    border: 7px solid lightgray;
+    width: 60px;
+    margin: 10px;
+}
+.image {
+    display: flex;
+}
+.image div {
+    margin: 40px;
+}
+h2 {
+    padding-top: 7px;
+    color: #464545;
+    font-weight: bold;
+}
+h3 {
+    font-size: 12px;
+    padding-top: 15px;
+    color: #464545;
+    font-style: italic;
+}
+h4 {
+    font-size: 12px;
```

Comment:
font-size of all header element cannot be 12 px. If we set all to 12px then, why do we need a different header at all.